users/tests.py

```diff
@@ -98,13 +98,13 @@ def owner_profile_as_user():
     return Apartment.objects.first().owner
 
 
-# @pytest.mark.django_db
-# def test_is_seeker(seeker_profile_as_user, owner_profile_as_user):
-#     assert seeker_profile_as_user.is_seeker is True
-#     assert owner_profile_as_user.is_seeker is False
+@pytest.mark.django_db
+def test_is_seeker(seeker_profile_as_user, owner_profile_as_user):
+    assert seeker_profile_as_user.is_seeker is True
```

Comment:
`is True` is redundant.
Read more about Truthy and Falsy values in python:
https://www.freecodecamp.org/news/truthy-and-falsy-values-in-python/
https://www.geeksforgeeks.org/truthy-vs-falsy-values-in-python/

Reply:
Changed it 💯